### DISCUSSION TITLE
Predicting dipole moment using NequIP

### DISCUSSION THREAD

**Discussion by nitbha007** [mir-group/nequip#350] [Q&A]
I am interested in utilizing NequIP for extending molecular dynamics (MD) simulations and predicting the dipole moment of the system. This objective can be similar to the approach presented in the PhysNet paper or FieldSchNet. While exploring related discussions on GitHub (https://github.com/mir-group/nequip/discussions/306), I found some relevant information. However, I would appreciate guidance on how to proceed further, particularly regarding integrating the dipole moment into the loss function and modifying the node features within the network.
 
To provide you with more context, my dataset consists of Cu13 and Cu55 nanoclusters with various small organic species (CH4, C2H6, H2O, N2, NOx, CH3OH) adsorbed on them. I have collected approximately 40k geometries per clusters by performing AIMD simulations using DFT-PBE and FHI-AIMS.
 
Thank you.

**Comment by Linux-cpp-lisp** [comment]
Hi @nitbha007 ,

Thanks for your interest in our work!

https://github.com/mir-group/nequip-example-extension contains examples of custom loss functions and modules in the NequIP framework, which should make it possible to implement various different approaches.  I think #306 contains more details on the conceptual setup.
